HelpMyStreetFE/HelpMyStreetFE/Views/Login/Login.cshtml

```diff
@@ -4,8 +4,15 @@
     Layout = "~/Views/Shared/_Layout.cshtml";
 }
 <script>
-    const configuration = { firebase: @Html.Raw(Json.Serialize(@Model.FirebaseConfiguration)) }
+    const configuration = { firebase: @Html.Raw(Json.Serialize(@Model.FirebaseConfiguration)) };
+
 </script>
+@if (!String.IsNullOrEmpty(Model.LoginURL))
```

Comment:
I'm not sure LoginURL is required here; we may be able to use the existing ReturnUrl parameter.